analyzer/codechecker_analyzer/buildlog/build_manager.py

```diff
@@ -63,6 +65,16 @@ def perform_build_command(logfile, command, context, keep_link, silent=False):
         with open(original_env_file, 'rb') as env_file:
             original_env = pickle.load(env_file)
 
+        # The env is loaded we do not need it anymore.
+        # Remove temporary directory.
+        try:
+            tmp_dir, _ = os.path.split(original_env_file)
+            shutil.rmtree(tmp_dir)
```

Comment:
This folder will only be removed only when we run the `CodeChecker log` command explicitly. There are a lot of other commands which will not remove this folder (`CodeChecker server`, `CodeChecker parser` etc.).

Reply:
Yes, I know it but thanks for pointing it out. I've changed the place of the cleanup. Maybe later we can do some optimization not to create this env file if it not necessary ( I think it is only used by the log and maybe the analyze subcommands).